clang/lib/Headers/ia32intrin.h

```diff
@@ -533,6 +533,8 @@ __rdtscp(unsigned int *__A) {
 /// \see __rdpmc
 #define _rdpmc(A) __rdpmc(A)
 
+;
```

Comment:
This is quite surprising; I'm guessing this is so that the previous comment does not associate with the function declared below, but this still feels pretty unclean.

Reply:
If it sounds reasonable to you, I can fix this with the same method used in the `clang/lib/Headers/amxcomplexintrin.h` file. Unfortunately changing the underlying comment parsing functionality would be a complicated change.

Reply:
Agreed, I think it will need to change at some point and I will see if we can get support for doing that work.